core/src/test/scala/unit/kafka/admin/DeleteTopicTest.scala

```diff
@@ -227,8 +227,10 @@ class DeleteTopicTest extends ZooKeeperTestHarness {
     }, s"Not all replicas for topic $topic are in states of either ReplicaDeletionSuccessful or OfflineReplica")
 
     // increase the partition count for topic
-    val topicCommandOptions = new TopicCommand.TopicCommandOptions(Array("--zookeeper", zkConnect, "--alter", "--topic", topic, "--partitions", "2"))
-    new ZookeeperTopicService(zkClient).alterTopic(topicCommandOptions)
+    val props = new Properties()
+    props.setProperty(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, TestUtils.getBrokerListStrFromServers(servers))
+    val adminClient = Admin.create(props)
+    adminClient.createPartitions(Map(topic -> NewPartitions.increaseTo(2)).asJava)
```

Comment:
There should be a "get" here to ensure that we actually create the partitions before shutting down, right?

Reply:
Good catch. Although this will return an API exception (can't increase partition number during topic deletion), we still need to wait for the completion.